client/src/Components/local/atoms/core-mapping/helper/map-controls.tsx

```diff
@@ -8,10 +8,43 @@ import AssetIcon from 'src/Components/local/asset-icon'
 import styles from '../styles.module.scss'
 import { GeomanControlProps } from '../types/props'
 
-const MapControls: React.FC<GeomanControlProps> = ({ onCreate }) => {
+const MapControls: React.FC<GeomanControlProps> = ({ onCreate, onShowLabels }) => {
   const map = useMap()
 
   const initMapListener = () => {
+    let layersVisible = true 
+    map.pm.Toolbar.createCustomControl({
+      name: 'showLayersText',
+      block: 'custom',
+      className: 'control-icon leaflet-pm-icon-text',
+      title: 'Show layers Text',
+      afterClick: () => {
+        onShowLabels(layersVisible)
+
```

Comment:
Hello @lilitkarapetyan - this next block is commented out.  Is it because you have found two ways of hiding/revealing text labels, and have implemented the "other one", but left this hear just in case we need it?

Reply:
Hello @ian, Yes